plugins/CoreHome/templates/_angularComponent.twig

```diff
@@ -0,0 +1,5 @@
+<{{ componentName }}
+    {% for key, value in componentParameters %}
+    {{ key }}="{{ value|e('html') }}"
```

Comment:
the componentName and key should be escaped here to be safe?

Reply:
👍 might be automatic w/ twig? can't remember, but will double check